static/temp/foods.json

```diff
@@ -0,0 +1,310 @@
+            "endUnit": "years"
+          },
+          {
+            "start": 14,
+            "startUnit": "years",
+            "end": 18,
+            "endUnit": "years"
+          },
+          {
+            "start": 19,
+            "startUnit": "years",
+            "end": 30,
+            "endUnit": "years"
+          },
+          {
+            "start": 31,
+            "startUnit": "years",
+            "end": 50,
+            "endUnit": "years"
+          },
+          {
+            "start": 51,
+            "startUnit": "years",
+            "end": 70,
+            "endUnit": "years"
+          }
+        ]
+      },
+      "tableHeaders": {
+        "allNutrients": [
+          "Protein",
+          "carbohydrate",
+          "water",
+          "vitamin",
+          "Fats",
+          "Fiber",
+          "Minerals"
+        ],
+        "valuePercent": "% Daily Value",
+        "rdi": {
+          "amount": 130,
+          "unit": "g"
+        }
+      },
+      "tableContent": [
+        {
+          "title": "Total Carbohydrate",
+          "amount": 4,
+          "amountUnit": "g",
+          "content": [
+            {
+              "nutrient": "Carbohydrate  2g",
+              "value": "17.67%",
+              "rdi": {
+                "amount": "130",
+                "unit": "g"
+              }
+            },
+            {
+              "nutrient": "Sugar  2g",
+              "valuePercent": "17.67%",
+              "rdi": { "amount": "15", "unit": "g" }
+            },
+            {
+              "nutrient": "Starch  2g",
+              "value": "17.67%",
+              "rdi": {
+                "amount": "12.6",
+                "unit": "g"
+              }
+            }
+          ]
+        },
+        {
+          "title": "Protein",
+          "amount": 11,
+          "amountUnit": "g",
+          "content": [{ "nutrient": "Protein 2g", "value": "1.61%", "rdi": "46g" }]
+        }
+      ]
+    },
+    "recipes": [
+      {
+        "name": "Banana Walnut Bread",
+        "description": "Flour, baking soda, ripe banana, vanilla extract",
+        "duration": {
+          "prepTime": {
+            "hours": 0,
+            "minutes": 10
+          },
+          "cookTime": {
+            "hours": 0,
+            "minutes": 20
+          },
+          "totalTime": {
+            "hours": 0,
+            "minutes": 30
+          }
+        },
+        "instructions": [
+          {
+            "step": 1,
+            "description": "Wash and cut your banana inside a neat bowl and mix thoroughly till it’s smoothly combined",
+            "videoUrl": ""
+          },
+          {
+            "step": 2,
+            "description": "Dice the spinach into smaller pieces and pour inside your mixed banana and stir thoroughly.",
+            "waitTime": {
+              "hours": 0,
+              "minutes": 10
+            }
+          }
+        ],
+        "reviews": [{ "1": 5, "2": 5, "3": 5, "4": 5, "5": 4 }],
```

Comment:
I think we should simplify so that front-end is not responsible for any calculations. Make this is

```json5
"reviews": {
    "total": 19,
    "rating": "4.5"
}
```

Reply:
done